### PR TITLE
FHIR-37120

### PR DESCRIPTION
**Pull request by EricOnFHIR**
For now remove the FHIR Server-based Content Sharing section due to the lack of implementation.  This concept can be revisited in subsequent version of FHIRcast.

(https://jira.hl7.org/browse/FHIR-37120)

### CODE REVIEW

**Comment by isaacvetter** [comment]
Hey @EricOnFHIR ,

Here are a few other locations in which content by reference is mentioned: 
* https://build.fhir.org/ig/HL7/fhircast-docs/artifacts.html -- remove Subcribe to container examples
* https://build.fhir.org/ig/HL7/fhircast-docs/4-7-hybrid-content-sharing.html -- all
* https://build.fhir.org/ig/HL7/fhircast-docs/SubscriptionTopic-container-update-topic.html -- all
* https://build.fhir.org/ig/HL7/fhircast-docs/SubscriptionTopic-container-update-resource.html - all

**Review by EricOnFHIR — COMMENTED** [review]
missed the update topic/resource, need to look at that tomorrow

**Comment by bvdh** [comment]
I've removed the SubscriptionTopic resources.
It also allows the FHIR version to be set to 4.0.1 instead of 4.3.0. I've updated the pull request accordingly.

**Review by EricOnFHIR — COMMENTED** [review]
Thank you Bas, then I believe the PR is ready to merge...

**Review by isaacvetter — COMMENTED** [review]
Eric, a diff against 2-10-1 in master yields not only a difference from a merge last night, but also the absence of the infonote/callout about resource ids.

File: input/pagecontent/2-3-Events.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ All fields available within an event's context SHALL be defined in a table where
...
 
-The key used for indicating a context change event's FHIR resource SHALL be the lower-case resourceType of the resource as defined in the [resource type valueset](http://build.fhir.org/valueset-version-independent-resource-types.html). For example, the [patient-open](3-3-1-Patient-open.html) event uses the `patient` key to signal the new Patient in context. Note that due to legacy reasons, some events deviate from this pattern. The resources to include are defined in the corresponding event definition in the [event catalog](3_Events.html).
+The key used for indicating a context change event's FHIR resource SHALL be the lower-case resourceType of the resource. The resources to include is defined in the corresponding event definition in the [standard event catalog](3_Events.html).
```
Comment (isaacvetter) [inline input/pagecontent/2-3-Events.md:58]:
```suggestion
The key used for indicating a context change event's FHIR resource SHALL be the lower-case resourceType of the resource as defined in the [resource type valueset](http://build.fhir.org/valueset-version-independent-resource-types.html). For example, the [patient-open](3-3-1-Patient-open.html) event uses the `patient` key to signal the new Patient in context. Note that due to legacy reasons, some events deviate from this pattern. The resources to include are defined in the corresponding event definition in the [event catalog](3_Events.html).

```

re-implement https://github.com/HL7/fhircast-docs/commit/3d78d56b7285173f8a890211fe15aff704e29e86

**Comment by isaacvetter** [comment]
Okay, I think I fixed it. Do you guys want to double-check?

**Comment by isaacvetter** [comment]
Merging b/c this PR is blocking others / keeps requiring rebasing.

**Pull request merged**